### PR TITLE
Suggested homepage text

### PR DESCRIPTION
**Pull request by JessMontgomery**
For you to amend as you see fit.

### CODE REVIEW

**Review by lawrennd — APPROVED** [review]
Looks great, thanks Jess! 

Let's also get Carl Henrik and Ferenc's eyes on it, they might like to add something to the ML section.

**Review by carlhenrikek — APPROVED** [review]
Very nice, thank you for writing this.

File: index.md [hunk 1]
```diff
@@ -5,5 +5,15 @@
...
+
+**Data**: The availability and quality of data influences the effectiveness of machine learning systems. There are opportunities now to develop machine learning methods that are better able to deal with messy or poor-quality data that systems often encounter in real-world environments. Data oriented programming, for example, offers a set of development methodologies to ensure system designers account for: what decisions are required, how they will be made, and how these interconnect within the system architecture. This creates the possibility of a more sophisticated form of auto ML, where full redployments of models are considered while analyzing the information dynamics of a complex automated decision-making system.
```
Comment (apaleyes) [inline index.md:14]:
```suggestion
**Data**: The availability and quality of data influences the effectiveness of machine learning systems. There are opportunities now to develop machine learning methods that are better able to deal with messy or poor-quality data that systems often encounter in real-world environments. Data oriented programming, for example, offers a set of development methodologies to ensure system designers account for: what decisions are required, how they will be made, and how these interconnect within the system architecture. This creates the possibility of a more sophisticated form of auto ML, where full redeployments of models are considered while analyzing the information dynamics of a complex automated decision-making system.
```

Comment (apaleyes) [reply index.md:14]: Just a tiny typo, but looks and reads great overall!

**Comment by lawrennd** [comment]
Going to merge now @fhuszar you can also make changes via pull request!

**Pull request merged**